sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { assert } from "chai";
+import assert from "assert";
```

Comment:
Let’s definitely not do this :)

Reply:
In general, we want to use `assert` from `chai` as much as possible, and we also want to stop using the `assert` package altogether.